docs/user/faq.rst

```diff
@@ -610,33 +610,85 @@ method, making it compatible with ``boto3``\'s
    pattern can be applied to any storage API that supports streaming directly
    from a file-like object.
 
-How do I consume a query string that has a JSON value?
-------------------------------------------------------
-Falcon defaults to treating commas in a query string as literal characters
-delimiting a comma separated list. For example, given
-the query string ``?c=1,2,3``, Falcon defaults to adding this to your
-``request.params`` dictionary as ``{'c': ['1', '2', '3']}``. If you attempt
-to use JSON in the value of the query string, for example ``?c={'a':1,'b':2}``,
-the value will get added to your ``request.params`` in a way that you probably
-don't expect: ``{'c': ["{'a':1", "'b':2}"]}``.
+How do I retrieve a JSON value from the query string?
+-----------------------------------------------------
+To retrieve a JSON-encoded value from the query string, Falcon provides the
+:meth:`~falcon.Request.get_param_as_json` method, an example of  which is given
+below:
+
+.. code:: python
+
+    import falcon
+
+
+    class LocationResource:
+
+        def on_get(self, req, resp):
+            places = {
+                'Chandigarh, India': {
+                    'lat': 30.692781,
+                    'long': 76.740875
+                },
+
+                'Ontario, Canada': {
+                    'lat': 43.539814,
+                    'long': -80.246094
+                }
+            }
+
+            coordinates = req.get_param_as_json('place')
+
+            for (key, value) in places.items():
+                if coordinates == value:
+                    place = key
+
+            resp.media = {
+                'place': place
```

Comment:
Probably not critical, but worth noting that `place` may be not initialized in case coordinates do not match (which can easily happen, provided the nature of `float`s :slightly_smiling_face: ).

The whole thing will likely explode with a `NameError` (untested)